### DISCUSSION TITLE
Choosing package manager (bun) on install

### DISCUSSION THREAD

**Discussion by GasparSukk** [tinacms/tinacms#4327] [Help]
Setting up new TinaCMS for my current Astro site.
I'm using [bun](https://bun.sh) package manager for this project.

On installation i can choose only between PNPM, Yarn and NPM. How to proceed?
Should i choose npm and change this later in some places?
Or can you add bun to package manager list?

**Comment by kldavis4** [comment]
You can choose another package manager and then migrate it to bun once tina is set up.
